hack/configapi/e2e/test.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Try to upload a file to S3 and then delete it using the configapi cli.
+# Check the file exists after uploading it.
+# Check the file does not exist after deleting it.
+
+readonly REGION="eu-central-1"
+readonly BUCKET="cdn-constellation-backend"
+# readonly BUCKET="resource-api-testing"
+
+readonly CLAIM_PATH="${TEST_TMPDIR}/maaClaim.json"
+cat << EOF > "$CLAIM_PATH"
+{
+  "x-ms-isolation-tee": {
+    "x-ms-sevsnpvm-tee-svn": 1,
+    "x-ms-sevsnpvm-snpfw-svn": 9,
+    "x-ms-sevsnpvm-microcode-svn": 116,
+    "x-ms-sevsnpvm-bootloader-svn": 4
+  }
+}
+EOF
+
+readonly DATE="2023-02-02-03-04"
+${PATH_CLI} --maa-claims-path "$CLAIM_PATH" --upload-date "$DATE" --region "$REGION" --bucket "$BUCKET"
+
+sleep .2 # Ensure cache invalidation has time to propagate.
+curl https://cdn.confidential.cloud/constellation/v1/attestation/azure-sev-snp/${DATE}.json 2> /dev/null | grep -qF "Not Found"
+if [[ $? -eq 0 ]]; then
+  echo "Missing uploaded version file: constellation/v1/attestation/azure-sev-snp/${DATE}.json"
+  exit 1
+fi
+
+sleep .2 # Ensure cache invalidation has time to propagate.
+curl https://cdn.confidential.cloud/constellation/v1/attestation/azure-sev-snp/${DATE}.json.sig 2> /dev/null | grep -qF "Not Found"
```

Comment:
There should also be a test to make sure this file was deleted

Reply:
At the end of the file: https://github.com/edgelesssys/constellation/pull/2293/files/4bd94cc96671917bc675bb7a1ab0929f790cd7e1#diff-b6254790b95806d7e38385eb2ddf86260d8742f38f3c3a0bdd7b27c5fc5fc264R49-R54

Reply:
This only checks if the `.json` file was deleted, not if the `.sig` file was deleted, no?